package.json

```diff
@@ -17,16 +17,22 @@
     "type": "git",
     "url": "git://github.com/xmldom/xmldom.git"
   },
-  "main": "./dom-parser.js",
+  "main": "domParser.js",
```

Comment:
Why change the source file name into camelCase?

I would generally favor keeping source file names in kebab-case, like the newer npm package names are generally in kebab-case. Just my (US) $0.02 worth.

Reply:
Reverted this.